src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml

```diff
@@ -0,0 +1,244 @@
+                        EpochStartTime0: 1110000,
+                        EpochKey1: "\xb0\xb1\xb2\xb3\xb4\xb5\xb6\xb7\xb8\xb9\xba\xbb\xbc\xbd\xbe\xbf",
+                        EpochStartTime1: 1110001,
+                        EpochKey2: "\xc0\xc1\xc2\xc3\xc4\xc5\xc6\xc7\xc8\xc9\xca\xcb\xcc\xcd\xce\xcf",
+                        EpochStartTime2: 1110002,
+                    }
+
+    - label:
+          "Step 0b: TH binds GroupIds 0x0001 and 0x0002 with GroupKeySetID
+          0x01a1 in the GroupKeyMap attribute list on GroupKeyManagement cluster
+          by writing the GroupKeyMap attribute with two entries as follows:"
+      cluster: "Group Key Management"
+      endpoint: 0
+      command: "writeAttribute"
+      attribute: "GroupKeyMap"
+      arguments:
+          value: [{ FabricIndex: 1, GroupId: G1, GroupKeySetID: 0x01a1 }]
+
+    - label: "Step 0c: TH sends a RemoveAllGroups command to DUT."
+      cluster: "Groups"
+      endpoint: endpoint
+      command: "RemoveAllGroups"
+
+    - label:
+          "Step 1a: TH sends a AddGroup command to DUT with the GroupID field
+          set to G1."
+      cluster: "Groups"
+      command: "AddGroup"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: G1
+              - name: "GroupName"
+                value: "Group1"
+      response:
+          values:
+              - name: "Status"
+                value: 0
+              - name: "GroupID"
+                value: G1
+
+    - label:
+          "Step 1b: TH sends a RemoveAllScenes command to DUT with the GroupID
+          field set to G1."
+      command: "RemoveAllScenes"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: G1
+      response:
+          values:
+              - name: "Status"
+                value: 0x00
+              - name: "GroupID"
+                value: G1
+    - label:
+          "Step 1c: TH sends a GetSceneMembership command to DUT with the
+          GroupID field set to G1."
+      command: "GetSceneMembership"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: G1
+      response:
+          values:
+              - name: "Status"
+                value: 0x00
+              - name: "GroupID"
+                value: G1
+              - name: "SceneList"
+                value: []
+
+    - label:
+          "Step 2a: TH sends a MoveToLevel command to DUT, with Level =0 and
+          TransitionTime =0 (immediate)"
+      cluster: "Level Control"
+      command: "MoveToLevel"
+      arguments:
+          values:
+              - name: "Level"
+                value: 0
+              - name: "TransitionTime"
+                value: 0
+              - name: "OptionsMask"
+                value: 1
+              - name: "OptionsOverride"
+                value: 1
+
+    - label: "Step 2b: TH reads the MinLevel attribute from the DUT"
```

Comment:
This doesn't match the test plan. 2b says:
TH reads OnOff attribute (On/Off cluster) from DUT

Reply:
discussed on slack - test plan is being changed.

Reply:
We are removing all cluster interaction between OnOff and Level Control in scenes recall, turns out it was not the expected behaviour, so every step/PICS related to OnOff has been removed. Will update Test Plan once the merges.